src/components/button/button.ts

```diff
@@ -39,6 +41,9 @@ export class MdButton {
   /** Whether a mousedown has occurred on this element in the last 100ms. */
   _isMouseDown: boolean = false;
 
+  /** Whether the ripple effect on click should be disabled. */
+  @Input('md-ripple-disabled') _rippleDisabled = false;
```

Comment:
I think this can just be `disableRipple` (without the `md-` as well). For now, it should probably use the `BooleanFieldValue` (though we're likely going to remove that soon)